src/cljam/io/csi.clj

```diff
@@ -0,0 +1,78 @@
+(ns cljam.io.csi
+  "Reader of a CSI format file."
+  (:require [cljam.io.util.bgzf :as bgzf]
+            [cljam.io.util.lsb :as lsb]
+            [cljam.io.util.bin :as util-bin]
+            [clojure.string :as cstr])
+  (:import java.util.Arrays
+           [java.io DataInputStream IOException]
+           [cljam.io.util.chunk Chunk]))
+
+(deftype Csi [n-ref min-shift depth bidx loffset]
```

Comment:
`CSI` sounds good to me bacause CSI is an acronym that stands for Coordinate Sorted Index.
https://guide.clojure.style/#CamelCase-for-protocols-records-structs-and-types